### DISCUSSION TITLE
Metadata fetching from Amazon?

### DISCUSSION THREAD

**Discussion by thehijacker** [crocodilestick/Calibre-Web-Automated#78] [Q&A] [answered]
I use heavily metadata fetching in Calibre, and since my goal is to ditch Calibre and switch all to CWA I hoped that metadata fetching will also works the same. But it doesn't. It doesn't find in most cases. And it doesn't search on Amazon even if it is selected. Can others meta data providers get disable and only Amazon (working) and Google is enabled?

**Comment by jmarmstrong1207** [comment, marked as answer]
This is more of an issue with stock calibre-web. Sometimes, Amazon blocks your IP for a bit if you reach a certain quota on the amount of metadata you're searching
